include/deal.II/lac/affine_constraints.h

```diff
@@ -547,24 +547,39 @@ class AffineConstraints : public Subscriptor
     right_object_wins
   };
 
+  /**
+   * Default constructor.
+   *
+   * This constructor sets up an object that stores all constraints eventually
+   * given to it. (This is in contrast to the following constructor that
+   * restricts which degrees of freedom we should care about -- for example,
+   * in a parallel context one would typically only store constraints
+   * for the locally relevant DoFs, see
+   * @ref GlossLocallyRelevantDof .)
+   * Consequently, this constructor creates internal data structures for
+   * <i>all</i> possible
+   * indices will be created, leading to memory consumption on every
```

Comment:
This was there before, but aren't there some stray words?
```suggestion
   * indices, leading to memory consumption on every
```